ui/tutorial-renderer/src/main/res/values/dimens.xml

```diff
@@ -5,4 +5,5 @@
     <dimen name="tutorial_page_inset_top">48dp</dimen>
     <dimen name="tutorial_page_inset_bottom">64dp</dimen>
     <dimen name="tutorial_page_live_share_anim_height">266dp</dimen>
+    <dimen name="tutorial_page_onboarding_anim_height">140dp</dimen>
```

Comment:
I turned on that ```show layout bounds``` mode you suggested and used that to find what looked right for this.

Reply:
before it was relying on the implicit size of the images, which had a height of 140. so you found the right height.

Reply:
the scales from `dp` -> `px` are:
`mdpi`: 1x
`hdpi`: 1.5x
`xhdpi`: 2x
`xxhdpi`: 3x

Reply:
so, looking at the dimensions of a mdpi image will reveal what size it is in DP

Reply:
only thing I would have done slightly different with this constant, is put it in the `styles_tutorial_onboarding.xml` file where the other onboarding styles are located

Reply:
Would it be worth moving there?

Reply:
yeah, just move it to the beginning of this file: https://github.com/CruGlobal/godtools-android/blob/develop/ui/tutorial-renderer/src/main/res/values/styles_tutorial_onboarding.xml

Reply:
it provides a bit of organization long term to prevent having to search the code base for things :)

Reply:
Maybe a dumb question, but should it be inside the style tag or outside?

Reply:
outside the `<style>` tag, but still inside the `<resources>` tag, you're still learning, so not dumb.

all the xml files within the `values` folder are treated the same, so you can mix and match resources in any file if you want to. It's really just conventions that dictate where things end up

Reply:
Makes sense thanks, Daniel! 🙂